src/lenten-chili.md

```diff
@@ -0,0 +1,41 @@
+# Lenten (Vegan) Chili
+
+- Prep time: 15 Minutes
+- Cook time: 45 Minutes
+- Serves: 4-8 people

```

Comment:
Use the emoji-variant:

```
- ⏲️ Prep time: 10 min
- 🍳Cook time: 30 min
- 🍽️ Servings: 4
```

Reply:
Thank you, I updated this and the other problem you pointed out and added an image.